files/dashboards/openshift-logging-dashboard.json

```diff
@@ -1708,18 +1798,233 @@
         "align": false,
         "alignLevel": null
       }
+    },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": null,
+      "fieldConfig": {
+        "defaults": {
+          "custom": {},
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          }
+        },
+        "overrides": []
+      },
+      "fill": 1,
+      "fillGradient": 0,
+      "gridPos": {
+        "h": 7,
+        "w": 8,
+        "x": 16,
+        "y": 49
+      },
+      "hiddenSeries": false,
+      "id": 41,
+      "legend": {
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 1,
+      "nullPointMode": "null",
+      "percentage": false,
+      "pluginVersion": "7.1.1",
+      "pointradius": 2,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "topk(10, fluentd_input_status_total_bytes_logged) by (path)",
```

Comment:
This is going to show more than 5 items because it maps the top 5 items at each
interval instead of across the whole graph. To accomplish what you're trying to
do you need to use the following query:

```
query_result(topk by(path)(5, avg_over_time(fluentd_input_status_total_bytes_logged[${__range_s}s])))
```

Reply:
@blockloop 
1)  `query_result(topk by(path)(10, avg_over_time(fluentd_input_status_total_bytes_logged[${__range_s}s])`  throwing error in while `(topk by(path)(10, avg_over_time(fluentd_input_status_total_bytes_logged[${__range_s}s])` works. 

2) I see several values in the Graph as shown in attached screenshot.  Is this query shows top 10 containers in the given range(time-interval) ? So in the timeline one will see 10 containers every range ? is this expected ? 
![charts_range_topk](https://user-images.githubusercontent.com/72593688/112005081-70204000-8b48-11eb-93b4-3ac8b7fe3a97.png)

Reply:
@jcantrill  Currently I am facing above issue so seeking help from Brett.